src/components/breadcrumbs/index.md.njk

```diff
@@ -10,8 +10,12 @@ copy: done
 layout: layout-pane.njk
 ---
 
+{% from "_example.njk" import example %}
```

Comment:
Can we import this globally somewhere rather than in each component file individually? Like the prototype kit does with Frontend components?